### PR TITLE
Edit post: Unable to specifically remove "Add caption" field for image

### PR DESCRIPTION
**Pull request by ushahidlee**
Fixes USH-777

### CODE REVIEW

File: libs/sdk/src/lib/services/media.service.ts [hunk 1]
```diff
@@ -2,13 +2,13 @@ import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { apiHelpers } from '../helpers';
 import { EnvLoader } from '../loader';
-import { MediaResponse } from '../models';
+// import { MediaResponse } from '../models';
 import { ResourceService } from './resource.service';
```
Comment (Angamanga) [inline libs/sdk/src/lib/services/media.service.ts:6]: Should MediaResponse be removed?

**Review by Angamanga — COMMENTED** [review]
When I test this out locally, the API complains about "The PATCH method is not supported for this route. Supported methods: GET, HEAD, PUT, DELETE." when trying to save a new caption/remove a caption. Does it work for you locally @ushahidlee?

File: apps/web-mzima-client/src/app/feed/feed.component.ts [hunk 2]
```diff
@@ -544,16 +544,8 @@ export class FeedComponent extends MainViewComponent implements OnInit {
...
-      if (oldPost) {
-        const updatedPost = _.cloneDeep(oldPost);
-        updatedPost.sets = _.cloneDeep(p.sets);
-        updatedPost.title = p.title;
-        updatedPost.content = p.content;
-        updatedPost.status = p.status;
-        updatedPost.post_content = p.post_content;
-        this.posts = this.posts.map((obj) => (obj.id === updatedPost.id ? updatedPost : obj));
-      }
+      const updatedPost = _.cloneDeep(p);
```
Comment (Angamanga) [inline apps/web-mzima-client/src/app/feed/feed.component.ts:547]: Very neat 🙌

**Comment by ushahidlee** [comment]
@Angamanga Yes this ticket can only be tested with a corresponding PR on platform 

https://github.com/ushahidi/platform/pull/4686

**Comment by Angamanga** [comment]
@ushahidlee Aahh, got it. Good time for me to set the API up then 🤓

**Pull request merged**